src/main/scala/arudkovskaia/hw_04/Task03.scala

```diff
@@ -0,0 +1,23 @@
+package arudkovskaia.hw_04
+
+import scala.io.Source
+
+/**
+ * 3. Repeat the preceding exercise with an immutable map.
+ */
+object Task03 {
+  def main(args: Array[String]): Unit = {
+    val source = Source.fromFile("src/main/scala/arudkovskaia/hw_04/myfile.txt", "UTF-8")
+    var wordsCount = scala.collection.immutable.Map[String, Int]()
```

Comment:
Please check the solution with "withDefaultValues()"

Reply:
I don't really remember what it is about..

Reply:
done with default values